interop/applier.go

```diff
@@ -177,3 +187,15 @@ func waitMinedWithTicker(ctx context.Context, b bind.DeployBackend, tx *types.Tr
 		}
 	}
 }
+
+func hexStringToBigInt(hexString string) *big.Int {
+	// Remove the "0x" prefix if it's present
+	if len(hexString) > 1 && hexString[:2] == "0x" {
+		hexString = hexString[2:]
+	}
+
+	bigInt := new(big.Int)
+	bigInt.SetString(hexString, 16)
+
+	return bigInt
+}
```

Comment:
nit: I think you might not need a helper function
 `new(big.Int).setBytes(common.FromHex(hexString))`

Reply:
oh nice, this is much better